plugins/ufm_log_analyzer_plugin/src/loganalyze/.pylintrc

```diff
@@ -4,3 +4,6 @@ disable=missing-function-docstring,
         missing-module-docstring,
         too-few-public-methods,
         logging-fstring-interpolation,
+
+[DESIGN]
```

Comment:
Please add a description of what this parameter means.

Reply:
It is the pylint config.... I don't think we need to explain pylint options.

But in this case, it will allow to each function have up to 20 local variables.